portfolio/src/main/webapp/script.js

```diff
@@ -176,18 +176,35 @@ google.charts.setOnLoadCallback(drawChart);
  * a pie chart, and draws it.
  */
 function drawChart() {
-  let data = new google.visualization.DataTable();
-  data.addColumn('string', 'Type');
-  data.addColumn('number', 'Number Watched');
-  data.addRows([
-      ['Anime', 10],
-      ['Disney', 8],
-      ['Action', 5]
-  ]);
-
-  let options = {'title': 'Movies I\'ve Watched',
-                 'width': 400,
-                 'height': 400};
-  let chart = new google.visualization.PieChart(document.getElementById("charts-id"));
-  chart.draw(data, options);
+  fetch('/charts').then(response => response.json()).then((data) => {
+    console.log(data);
+
+    let dataTable = new google.visualization.DataTable();
+    dataTable.addColumn('string', 'Gender');
+    dataTable.addColumn('number', 'Very Favorable');
+    dataTable.addColumn('number', 'Somewhat Favorable');
+    dataTable.addColumn('number', 'Somewhat Unfavorable');
+    dataTable.addColumn('number', 'Very Unfavorable');
+    dataTable.addColumn('number', 'Heard Of; No Opinion');
+    dataTable.addColumn('number', 'Never Heard of');
+    
+    Object.keys(data).forEach((gender) => {
+      dataTable.addRow([
+          gender,
+          data[gender][0],
```

Comment:
Maybe it'd be nicer if you called this once before addRow, like:

let gender_stats = data[gender];